source/blog/2018-10-05-the-ember-times-issue-67.md

```diff
@@ -1,18 +1,30 @@
 ---
 title: The Ember Times - Issue No. XX
-author: the crowd
+author: Ryan Mark, Amy Lam
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
 alias : "blog/xxxx/xx/xx-the-ember-times-issue-XX.html"
 responsive: true
 ---
 
-<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
+Hello, pleased to eat you, Emberistas! -Dracula 🧛‍♂️
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Fall is here and so is Hacktoberfest! We're also sharing... 
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Hacktoberfest 🎃🍻👨‍💻](https://hacktoberfest.digitalocean.com/)
+
+Trick or treat! The fifth annual Hacktoberfest is here! Support open source and earn a limited edition T-shirt by making **five** pull requests this month to any public repos...why not all Ember ones?! 
+
+Looking for something to work on? Pop into the [#hacktoberfest](https://discordapp.com/channels/480462759797063690/496453502298750988) channel on Discord and ask away with questions! Here are some issues to get you started:
```

Comment:
I think [Ember Styleguide](https://github.com/ember-learn/ember-styleguide/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest) and the [Website](https://github.com/emberjs/website/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest) also have a couple of new Hacktoberfest issues, can we also add them to the list?

Reply:
Added and alphabetized